site/docs/v1/tech/guides/basic-registration-forms.adoc

```diff
@@ -0,0 +1,162 @@
+
+Navigate to [breadcrumb]#Tenants -> Your Tenant# and then select the [breadcrumb]#Email# tab. Scroll to the [breadcrumb]#SMTP settings# section. Configure this with your SMTP server information. If you are testing locally, you can use a mock email server like https://mailcatcher.me/[MailCatcher].
+
+image::guides/basic-registration-forms/tenant-smtp-settings.png[Tenant SMTP settings,width=1200,role=bottom-cropped]
+
+Ensure a test email is delivered. If you have any issues, follow the steps in the link:/docs/v1/tech/admin-guide/troubleshooting#troubleshooting-email[email troubleshooting documentation].
+
+Save the configuration.
+
+=== Configuring Basic Registration
+
+To set up Basic Registration, navigate to the Applications tab, and select the "Edit" icon to open the edit page.
+
+Then navigate to the [breadcrumb]#Registration# tab. Scroll down to the "Self Service Registration" section. Toggle the "Enabled" switch on.
+
+The "Type" radio button should have "Basic" selected. Leave it on this setting.
+
+You can toggle on the "Verify Password" switch to ask the user to enter their password twice when creating an account to ensure it is captured correctly.
+
+Leave the "Login type" radio button set on "Email".
+
+Then set the "Registration Fields" as follows:
+
+- First name: "Enabled" and "Required" on.
+- Last name: "Enabled" on and "Required" off.
+
+Set all the other fields to "Enabled" off.
+
+The settings should now look like this:
+
+image::guides/basic-registration-forms/basic-registration-settings.png[Basic Registration Settings,width=1200,role=bottom-cropped]
+
+=== Configuring Registration Verification Settings
+
+You can configure additional settings around registration verification. Scroll up to the [breadcrumb]#Registration Verification Settings# section, just above the "Registration Fields" section.
+
+Here you can specify if a registration verification email should be sent to the user. Toggling the "Verify registrations" switch on will reveal some further options.
+
+You can choose the email template to send for registration verification. Choose the "Registration Verification" template.
+
+For the "Verification Strategy" option, you can choose to send a link in the verification email to the user, or to send a short code that they can type in the registration form to verify their account.
+
+If you choose to send a short code, you must also set the "Unverified behaviour" option to "Gated". Note that the "Gated" behaviour option, and thus the short code option, is only available with a valid license key. Please visit link:/pricing[our pricing page] to review paid edition options and buy a license.
+
+Your verification settings should now look like this:
+
+image::guides/basic-registration-forms/registration-verification-settings.png[Registration Verification Settings,width=1200,role=bottom-cropped]
```

Comment:
I think you want to use a link strategy, since this is aimed at users without a paid edition?